components/src/core/three-liner/three-liner.component.scss

```diff
@@ -0,0 +1,19 @@
+$transition: all 300ms;
+
+.pxb-three-liner {
+    display: flex;
+}
+
+.pxb-three-liner-title {
+    font-size: 30px;
+    transition: $transition;
+}
+.pxb-three-liner-subtitle {
+    font-size: 16px;
+    transition: $transition;
+}
+.pxb-three-liner-info {
+    font-size: 14px;
```

Comment:
These font sizes should be in rem to respond to system font size

Reply:
Good catch; now using rems.